code/game/objects/items/storage/storage.dm

```diff
@@ -56,6 +56,7 @@
 	var/allow_quick_gather
 	///whether this object can change its drawing method (pouches)
 	var/allow_drawing_method
+	///0 = will open the inventory if you click on the storage container, 1 = will draw from the inventory if you click on the storage container
 	var/draw_mode = 0
```

Comment:
out of scope of this pr, but since there are only 2 draw modes (as far as I am aware) this could probably just be a clickdraw flag of some kind.